Library/Homebrew/test/cmd/autoremove_spec.rb

```diff
@@ -5,4 +5,44 @@
 
 describe "brew autoremove" do
   it_behaves_like "parseable arguments"
+
+  describe "integration test" do
+    let(:testball1) { Formula["testball1"] }
+    let(:testball2) { Formula["testball2"] }
+
+    before do
+      setup_test_formula "testball1" # will get installed as dependency
+      install_test_formula "testball2", <<-RUBY
+        depends_on "testball1"
+      RUBY
+    end
+
+    it "only removes unused dependencies", :integration_test do
+      expect(testball1.any_version_installed?).to be true
+      expect(testball2.any_version_installed?).to be true
+
+      # When there are no unused dependencies
+      expect { brew "autoremove" }
```

Comment:
Ideally I'd say only have one `brew` invocation in this test, even if it reduces the coverage a bit, as it's super slow.

Reply:
On my machine there's only like a 5 second difference between running one and two tests and I'm still using this old iMac. Not sure how important that 5 seconds is. But yeah, I can take one out if you want.

Reply:
I think important enough to omit tbh, sorry!

Reply:
Got it. Changed it to only one `brew` invocation in the integration test.